about/index.md

```diff
@@ -12,7 +12,7 @@ developing add-on modules to offer unique capabilities. Forseti’s core modules
 provide a foundation that others can build upon.
 
 Get started with
-[Forseti Security]({% link _docs/latest/configure/configuring-forseti.md %}).
+[Forseti Security]({% link _docs/latest/configure/forseti/index.md %}).
```

Comment:
Please insert a line between the page title and the first heading: "---"

Reply:
Updated.